pandas/core/groupby/groupby.py

```diff
@@ -1753,8 +1751,8 @@ def pre_processor(vals):
 
             return vals, inference
 
-        def post_processor(vals, inference):
-            # type: (np.ndarray, Optional[Type]) -> np.ndarray
+        def post_processor(vals: np.ndarray, inference: Optional[Type]) -> \
```

Comment:
We don't use explicit breaks like this anywhere else in the codebase so if you could make this implicit would be preferable